cmd/buildah/from.go

```diff
@@ -65,12 +71,18 @@ func fromCmd(c *cli.Context) error {
 	if len(args) > 1 {
 		return errors.Errorf("too many arguments specified")
 	}
-	image := args[0]
 
+	image := args[0]
 	registry := DefaultRegistry
 	if c.IsSet("registry") {
 		registry = c.String("registry")
 	}
+
+	credsContext, err := util.ContextFromGlobalOptions(c, "")
+	if err != nil {
+		return errors.Errorf("authentication failed for image [%v]", err)
```

Comment:
authentication didn't fail here, we just got some error grabbing stuff to build up a context object. Maybe `error building system context`